scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -52,12 +74,37 @@ public static BuildTargetResult UpdateBuild(BuildTargetContext c)
 
         // Moving PrepareTargets.RestorePackages after PackagePkgProjects because managed code depends on the
         // Microsoft.NETCore.App package that is created during PackagePkgProjects.
-        [Target(nameof(PrepareTargets.Init), nameof(CompileCoreHost), nameof(PackagePkgProjects), nameof(PrepareTargets.RestorePackages), nameof(CompileStage1), nameof(CompileStage2))]
+        [Target(nameof(PrepareTargets.Init), nameof(CompileCoreHost), nameof(PackagePkgProjects), nameof(CompileTargets.GenerateStubHostPackages), nameof(PrepareTargets.RestorePackages), nameof(CompileStage1), nameof(CompileStage2))]
         public static BuildTargetResult Compile(BuildTargetContext c)
         {
             return c.Success();
         }
 
+        [Target]
+        public static BuildTargetResult GenerateStubHostPackages(BuildTargetContext c)
+        {
+            string currentRid = GetRuntimeId();
+            var buildVersion = c.BuildContext.Get<BuildVersion>("BuildVersion");
+
+            foreach (var hostPackageId in HostPackages)
+            {
+                foreach (var rid in NetCoreAppRids)
+                {
+                    if (! rid.Equals(currentRid))
+
+                    {
+                        CreateDummyRuntimeNuGetPackage(
```

Comment:
Unfortunately given

1. That Microsoft.NETCore.App is built during this same build
2. That we can only build 1 runtime specific version of DotNetHost during the build
3. NuGet wants to restore all runtimes for a project.json

We have to generate these 'dummy' packages.  I've tried to change NuGet with https://github.com/NuGet/Home/issues/2534, but that doesn't seem to be going anywhere. Hopefully in RTM we move DotNetHost and Microsoft.NETCore.App packages out of dotnet/cli, then we won't need to do this anymore.